app/controllers/application_controller.rb

```diff
@@ -45,6 +45,6 @@ def check_entitled(entitlement)
   end
 
   def pundit_user
-    Thread.current[:user]
+    UserContext.current_user_context
```

Comment:
In PR https://github.com/RedHatInsights/catalog-api/pull/655 which introduced this change the `Thread.current[:user]` call was added to `app/services/catalog/user_capabilities.rb`.

Should that call be updated to use `UserContext.current_user_context` as well?

Reply:
Done